.github/workflows/deploy.yml

```diff
@@ -40,23 +54,29 @@ jobs:
 
       - uses: actions/checkout@v2
 
+      - uses: docker/setup-qemu-action@v1
+        name: Set up QEMU
+        if: ${{ matrix.cibw_archs == 'aarch64' || matrix.cibw_archs == 'ppc64le' }}
+
       - uses: actions/setup-python@v2
         with:
           python-version: '3.7'
 
       - name: Install cibuildwheel
         run: |
           python -m pip install --upgrade pip
-          python -m pip install cibuildwheel==1.5.5
+          python -m pip install cibuildwheel==2.1.1
 
       - name: Build wheels
         run: |
-          python -m cibuildwheel --output-dir wheelhouse
+          python -m cibuildwheel --output-dir dist
+        env:
+          CIBW_ARCHS: ${{ matrix.cibw_archs }}
 
       - uses: actions/upload-artifact@v2
         with:
-          name: ${{ runner.os }}-wheels.zip
-          path: ./wheelhouse/*.whl
+          name: ${{ matrix.os }}-${{ matrix.cibw_archs }}-wheel
```

Comment:
Do we still need `.zip` at the end?

Reply:
Also out of curiosity, is the name of this zipped file something that needs to follow a strict convention for PyPI? Or does the naming just need to be internally consistent?

Reply:
As above :slightly_smiling_face:

Reply:
Regarding the naming, since we arent uploading the artifacts directly from the zip (the names clobber one another in the previous format, hence the added architecture builds). We will unzip all the wheels into a common directory and upload from there, so we should be fine.